spec/features/guest/show_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+describe 'As a visitor' do
+  describe 'When I visit a guests show page' do
+    let!(:mtn_1) {Hotel.create!(name: "Sun Valley Main", location: "North Slopes")}
+    let!(:mtn_2) {Hotel.create!(name: "Sun Valley Private", location: "West Slopes")}
+    let!(:tori) {Guest.create!(name: "Tori", nights_stayed: "7")}
+    let!(:penthouse) {tori.rooms.create!(rate: 9998, suite: "Penthouse", hotel_id: mtn_1.id)}
+    let!(:cabin) {tori.rooms.create!(rate: 8798, suite: "Private Cabin", hotel_id: mtn_2.id)}
+    let!(:suite) {tori.rooms.create!(rate: 998, suite: "Suite", hotel_id: mtn_1.id)}
```

Comment:
Nice object setup for your tests - generally I suggest creating a few more Guest objects, to play with and make more scenarios with.